src/coreComponents/events/EventManager.cpp

```diff
@@ -227,15 +227,29 @@ bool EventManager::run( DomainPartition & domain )
 void EventManager::outputTime() const
 {
   // The formating here is a work in progress.
-  GEOS_LOG_RANK_0( GEOS_FMT( "\n"
-                             "------------------- TIMESTEP START -------------------\n"
-                             "    - Time:       {}\n"
-                             "    - Delta Time: {}\n"
-                             "    - Cycle:      {}\n"
-                             "------------------------------------------------------\n\n",
-                             units::TimeFormatInfo::fromSeconds( m_time ),
-                             units::TimeFormatInfo::fromSeconds( m_dt ),
-                             m_cycle ));
+  GEOS_LOG_RANK_0( "\n------------------- TIMESTEP START -------------------" );
+  if( m_maxTime < std::numeric_limits< real64 >::max() )
+  {
+    GEOS_LOG_RANK_0( GEOS_FMT( "    - Time:       {} out of {} ({}%)",
+                               units::TimeFormatInfo::fromSeconds( m_time ),
+                               units::TimeFormatInfo::fromSeconds( m_maxTime ),
+                               fmt::format( "{:.{}f}", 100.0 * m_time / m_maxTime, 0 ) ) );
```

Comment:
I think this needs to include `m_minTime` for example:

- `m_time` might be negative
- `m_maxTime` might be zero.

Reply:
Could the text here be extremely long? For example simulation up to 1000 years.

Reply:
I removed `m_maxTime` output and included `m_minTime`
For https://github.com/GEOS-DEV/GEOS/issues/3155 - I agree it is confusing and I don't really like the mixed format with years months etc going together to be honest
Would just report in years or in months or in days or etc...
@dkachuma feel free to modify this PR directly, I don't mind at all

Reply:
Pushed a change to `Units.cpp`. @MelReyCG Can you please take a look.

Reply:
Hi there,
The mixed format is really helping depending on the use case:
- the h/m/s time is useful to quickly read and understand the simulation time,
- the percentage you added is a good idea to quickly see the simulation progression,
- the second format is ideal and for sub-second simulation times and debugging purposes.

The problem is that sometimes, we need all formats with the same log, and we can have simulations that can take 24h+, so changing the desired format in the `xml` is not an option.

We could separate the second format:
```
------------------- TIMESTEP START -------------------
    - Time:       11h31m46s out of 12h00m00s  (96%)
                  ( 41506.95883192398 s / 43200 s )
    - Delta Time: 00h01m40s (100 s)
    - Cycle:      2167
------------------------------------------------------
```

... or we could separate the ending time:
```
------------------- TIMESTEP START -------------------
    - Time:       11h31m46s ( 41506.95883192398 s )
                  96% out of 12h00m00s ( 43200 s )
    - Delta Time: 00h01m40s (100 s)
    - Cycle:      2167
------------------------------------------------------
```

Reply:
Hi again @paveltomin,
Here, users have voted for the first alternative I posted, where formats are not mixed within the same line :)

```
------------------- TIMESTEP START -------------------
    - Time:       11h31m46s out of 12h00m00s  (96%)
                  ( 41506.95883192398 s / 43200 s )
    - Delta Time: 00h01m40s (100 s)
    - Cycle:      2167
------------------------------------------------------
```

If you'd like to, I can push what is needed in your PR to set it up.

Reply:
Yes, please go ahead and do that. Thank you!